notebooks/official/feature_store/online_feature_serving_and_fetching_bigquery_data_with_feature_store.ipynb

```diff
@@ -88,20 +88,6 @@
         "- Use the online server to fetch feature values for online prediction."
```

Comment:
<p>Please mark this section as "Preview", because data sync will go GA untill BQ --&gt; BT exporter goes GA.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2495/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Reply:
Unable to tell which section is called out, but added a Note above "Start manual Sync section"